### DISCUSSION TITLE
Appveyor Cache

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#610] [Ideas]
Appveyor allows saving cache for build. https://www.appveyor.com/docs/build-cache/

Probably we can do that for nuget package. 

Any thought? @Flow-Launcher/team

**Comment by jjw24** [comment]
Maybe whoever can do a pr to test out the build time difference. I haven't read the above link yet but this is a good idea

**Reply by taooceros** [reply]
Yeah let's see

**Reply by JohnTheGr8** [reply]
We have very few dependencies (compared to other projects, that is) and they take a few ms each to restore. Caching might shave off a couple seconds or so total from build time... I don't see why you should waste time on this @taooceros

**Reply by taooceros** [reply]
Yeah you are right🤯
